src/brnd/brnd-sportcenter-today.js

```diff
@@ -0,0 +1,230 @@
+ * @param {object} props.templateRootStyle - The template root style.
+ * @param {Function} props.getTitle - Function to get title for event.
+ * @returns {JSX.Element} - The component.
+ */
+function BrndSportcenterToday({
+  content,
+  bookings,
+  templateClasses = [],
+  templateRootStyle = {},
+  getTitle,
+}) {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const { title = "" } = content;
+
+  /** Imports language strings, sets localized formats. */
+  useEffect(() => {
+    dayjs.extend(localizedFormat);
+  }, []);
+
+  /**
+   * Capitalize the datestring, as it starts with the weekday.
+   *
+   * @param {string} s The string to capitalize.
+   * @returns {string} The capitalized string.
+   */
+  const capitalize = (s) => {
+    return s.charAt(0).toUpperCase() + s.slice(1);
+  };
+
+  // Sort events by datetime and filter away events that are done.
+  const getSortedBookings = (data) => {
+    const now = dayjs();
+
+    return data
+      .filter((e) => {
+        const startDate = dayjs(e.startTime * 1000);
+
+        return e.endTime > now.unix() && startDate.date() === now.date();
+      })
+      .sort((a, b) => a.startTime - b.startTime);
+  };
+
+  useEffect(() => {
+    const dateAndTimeInterval = setInterval(
+      () => setCurrentDate(new Date()),
+      1000
+    );
+    return () => clearInterval(dateAndTimeInterval);
+  }, []);
+
+  return (
+    <Wrapper
+      className={`template-brnd brnd-sportcenter-today ${templateClasses.join(
+        " "
+      )}`}
+    >
+      <Header className="header" style={templateRootStyle}>
+        <HeaderDate className="header-date">
+          {currentDate &&
+            capitalize(dayjs().locale(localeDa).format("dddd D. MMMM HH:mm"))}
+        </HeaderDate>
+      </Header>
+
+      <Title className="title">{title}</Title>
+
+      <Content className="schedule">
+        <ContentItemsWrapper className="schedule-header">
+          <ContentHeaderItem className="schedule-header-item">
+            <FormattedMessage id="when" defaultMessage="Tid" />
+          </ContentHeaderItem>
+          <ContentHeaderItem className="schedule-header-item">
+            <FormattedMessage id="booking-by" defaultMessage="Booket af" />
+          </ContentHeaderItem>
+          <ContentHeaderItem className="schedule-header-item">
+            <FormattedMessage id="facility" defaultMessage="Facilitet" />
+          </ContentHeaderItem>
+          <ContentHeaderItem className="schedule-header-item">
+            <FormattedMessage id="activity" defaultMessage="Aktivitet" />
+          </ContentHeaderItem>
+          <ContentHeaderItem className="schedule-header-item">
+            <FormattedMessage id="remarks" defaultMessage="Bemærkning" />
+          </ContentHeaderItem>
+        </ContentItemsWrapper>
+        <ContentItemsWrapper className="schedule-rows">
+          {bookings?.length > 0 &&
+            getSortedBookings(bookings).map((entry) => {
```

Comment:
Why bind returnFragment to a variable and return it afterwards?
```suggestion
     getSortedBookings(bookings).map((entry) => (
       <Fragment>
       
       </Fragment>
     )
```